src/containers/ImportResources/ImportResources.js

```diff
@@ -342,6 +355,27 @@ export class ImportResources extends Component {
                   defaultMessage: 'ServiceAccount (optional)'
                 })}
               />
+              <Dropdown
+                helperText={intl.formatMessage({
+                  id: 'dashboard.importResources.method.helperText',
+                  defaultMessage:
+                    "If any of the resources being imported use 'generateName' rather than 'name' in their metadata, select 'create' so they can be imported correctly."
+                })}
+                id="import-method"
+                initialSelectedItem={{ id: initialMethod, text: initialMethod }}
+                items={[
+                  { id: 'apply', text: 'apply' },
```

Comment:
The text something the should be translatable

Reply:
Discussed on slack, we'll leave this as they are for now since they map directly to the `kubectl apply` and `kubectl create` commands. We can revisit translation based on user feedback.